src/Umbraco.Core/Services/DataTypeUsageService.cs

```diff
@@ -0,0 +1,25 @@
+using Umbraco.Cms.Core.Persistence.Repositories;
+using Umbraco.Cms.Core.Scoping;
+
+namespace Umbraco.Cms.Core.Services;
+
+public class DataTypeUsageService : IDataTypeUsageService
+{
+    private readonly IDataTypeUsageRepository _dataTypeUsageRepository;
+    private readonly ICoreScopeProvider _scopeProvider;
+
+    public DataTypeUsageService(
+        IDataTypeUsageRepository dataTypeUsageRepository,
+        ICoreScopeProvider scopeProvider)
+    {
+        _dataTypeUsageRepository = dataTypeUsageRepository;
+        _scopeProvider = scopeProvider;
+    }
+
+    /// <inheritdoc/>
```

Comment:
Remove as it doesn't add any information
```suggestion
```

Reply:
I think it makes sense to inherit doc on the concrete implementations since it'll inherit the message from the interface, this way you'll still have context if you new it up in for instance a test.

Also, it'll give a suggestion to add the doc if I remove it :D

Reply:
Ahh yes, that's true!